src/snowcli/plugins/nativeapp/commands.py

```diff
@@ -99,29 +99,29 @@ def app_bundle(
 def app_run(
     version: Optional[str] = typer.Option(
         None,
-        help=f"""The identifier or 'version string' of the version you would like to create a version and/or patch for.
-        If not specified, the version from manifest.yml will be used.""",
+        help=f"""The identifier or version name of the version of an existing application package from which you want to create an application instance.
+        The application and application package names are determined from the project definition file.""",
     ),
     patch: Optional[str] = typer.Option(
         None,
         "--patch",
-        help=f"""The patch number you would like to create for an existing version.
-        If not specified, the patch from manifest.yml will be used if it is present in the file. Otherwise, Snowflake will auto-generate the patch number.""",
+        help=f"""The patch number, together with `--version` name of an existing application package, from which you want to create an application instance.
```

Comment:
The wording is a little tricky here. How about

> The patch number under the given `--version` that should be used to create the application instance.

Reply:
I'm not sure what it means for a patch number to be "under" a version.

Reply:
For the patch, it can only exist if a version already exists. And there can be many different patches under one version of an app package. I debated a few alternatives:
"patch associated with the `--version`"
"patch together with the `--version`"
but they do not seem to convey the many to one relationship between patches and a version. Hence the phrasing.